src/Pages/yoonseonghwang/Main/Main.scss

```
@@ -0,0 +1,319 @@
+.mainPage1 {
+  background-color: #fafafa;
+  .junkLine {
```

```suggestion
.mainPage {
  background-color: #fafafa;
  
  .junkLine {
```
- 여기도 클래스명 다시 지어주세요!
- 새로운 셀렉터 앞에서는 한줄 띄워주시는게 좋습니다